aggregate_test.go

```diff
@@ -724,6 +724,2342 @@ func Test_Aggregate(t *testing.T) {
+						Version:      "2.1.1",
+						WIP:          false,
+					},
+				},
+				[]Bundle{
+					Bundle{
+						Changelogs: []Changelog{
+							Changelog{
+								Component:   "vault",
+								Description: "Vault version updated.",
+								Kind:        "changed",
+							},
+						},
+						Components: []Component{
+							Component{
+								Name:    "vault",
+								Version: "0.7.3",
+							},
+						},
+						Dependencies: []Dependency{},
+						Deprecated:   false,
+						Name:         "cert-operator",
+						Time:         time.Date(2017, time.October, 26, 16, 53, 0, 0, time.UTC),
+						Version:      "0.1.0",
+						WIP:          false,
+					},
+					Bundle{
+						Changelogs: []Changelog{
+							Changelog{
+								Component:   "flannel",
+								Description: "Flannel version updated.",
+								Kind:        "changed",
+							},
+						},
+						Components: []Component{
+							Component{
+								Name:    "flannel",
+								Version: "0.9.0",
+							},
+						},
+						Dependencies: []Dependency{},
+						Deprecated:   false,
+						Name:         "flannel-operator",
+						Time:         time.Date(2017, time.October, 27, 16, 21, 0, 0, time.UTC),
+						Version:      "0.1.0",
+						WIP:          false,
+					},
+					Bundle{
+						Changelogs: []Changelog{
+							Changelog{
+								Component:   "component",
+								Description: "Put your description here",
+								Kind:        "changed",
+							},
+						},
+						Components: []Component{
+							Component{
+								Name:    "calico",
+								Version: "3.0.2",
+							},
+							Component{
+								Name:    "containerlinux",
+								Version: "1632.3.0",
+							},
+							Component{
+								Name:    "docker",
+								Version: "17.09.0",
+							},
+							Component{
+								Name:    "etcd",
+								Version: "3.3.1",
+							},
+							Component{
+								Name:    "coredns",
+								Version: "1.0.6",
+							},
+							Component{
+								Name:    "kubernetes",
+								Version: "1.9.2",
+							},
+							Component{
+								Name:    "nginx-ingress-controller",
+								Version: "0.11.0",
+							},
+						},
+						Dependencies: []Dependency{},
+						Deprecated:   false,
+						Name:         "kvm-operator",
+						Time:         time.Date(2018, time.March, 13, 12, 30, 0, 0, time.UTC),
+						Version:      "2.1.2",
+						WIP:          true,
+					},
+				},
+				//--------------------------------------------------------------------------------------------------------------
+				//--------------------------------------------------------------------------------------------------------------
+				//--------------------------------------------------------------------------------------------------------------
+				//--------------------------------------------------------------------------------------------------------------
+				//--------------------------------------------------------------------------------------------------------------
```

Comment:
Not sure what this means. We may want to provide some information here or remove it.

Reply:
Sorry, this was for myself when I was adding second portion and I had to go back and forth in the file to find corresponding bundles. Will remove :)